mlflow/ml-package-versions.yml

```diff
@@ -337,3 +337,15 @@ spark:
     run: |
       # TODO: Add datasource autologging tests
       find tests/spark_autologging/ml -name 'test*.py' | xargs -L 1 pytest --large
+
+pip:
```

Comment:
@harupy If we don't plan to explicitly require `pip >= 20.1` in `setup.py`, then we don't need to worry about MLR / Microsoft. However, how do we plan to handle the case where the user's installed pip version is less than the minimum? We could raise an exception if the user passes a file path to `pip_requirements` if `pip < 20.1`.

If MLR / MSFT agrees that it's okay, my preference would be to add `pip` to `setup.py`.